_posts/2024-07-22-pyopensci-at-scipy-2024.md

```diff
@@ -0,0 +1,463 @@
+---
+layout: single
+title: "pyOpenSci a Recap of @ SciPy 2024 Meeting - Tacoma, Washington"
+excerpt: "This year I gave a talk and lead a packaging workshop at the SciPy 2024 conference in Tacoma Washington. Learn more about the SciPy meeting and how exciting it was for pyOpenSci to be there this year!"
```

Comment:
are you thinking of the actual blog post page? this is what I see when I go to the main blog page:

<img width="1160" alt="Screenshot 2024-08-02 at 5 38 49 AM" src="https://github.com/user-attachments/assets/17c58d88-717b-4111-96da-05aba387bc20">

Reply:
so there's no author information

(also, I just noted that the alignment is off between the title and the body of the preview text. should I file an issue for that?)

Reply:
that alignment is intentional. I just redid that page this week with the events pr. 
 
the issue that i think you should file is related to wanting the author to be listed on the blog listing page. This topic is  out of scope for this pr because this pr is only a new blog post.  

Being super specific about where information is missing (in context of a specific PR) is important to avoid scope creep in a single pr and conversations then straying from the pr that we are trying to merge.

Feel free to open an issue (and a pr?) related to the blog landing page and please include screenshots and make it specific if you want someone else to fix it so we can tag it with help wanted!